tests/CMakeLists.txt

```diff
@@ -178,7 +178,7 @@ if(BUILD_GENIVI)
                      src/logger.cc
                      tests/rvisotaclient_test.cc)
     add_executable(aktualizr_test_rvi_client ${TEST_RVI_SRC})
-    target_link_libraries(aktualizr_test_rvi_client rvi ${TEST_LIBS} gtest)
+    target_link_libraries(aktualizr_test_rvi_client ${TEST_LIBS} gtest)
```

Comment:
Ah OK. The CMakeLists.txt in rvi_lib includes the line "target_include_directories(rvi PUBLIC include)" which automatically includes the RVI public interface headers in any target that links to it.  I can see that this isn't what you want in this case though :)